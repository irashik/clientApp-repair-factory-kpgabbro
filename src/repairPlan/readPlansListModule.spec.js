


import React from 'react';
import { shallow, render } from 'enzyme';

import ReadPlansListModule from './readPlansListModule';



describe('ReadPlanListModule', () => {

    test('snapShot correctly', () => {

        const component = shallow(<ReadPlansListModule />);
        expect(component).toMatchSnapshot();

    });
});