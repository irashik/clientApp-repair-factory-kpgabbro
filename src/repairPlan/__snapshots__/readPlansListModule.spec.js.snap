// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadPlanListModule snapShot correctly 1`] = `
<div>
  <h2>
    Нет записей
  </h2>
</div>
`;
